miso-web/src/main/webapp/scripts/hot.js

```diff
@@ -1166,7 +1166,31 @@ var HotUtils = {
       table.setCellMeta(i, sortColIndex, 'sortFunction', sortFunction);
     }
     table.sort(sortColIndex);
-  }
+  },
+  
+  getPlatformType: function(value) {
+    return Utils.array.maybeGetProperty(Utils.array.findFirstOrNull(function(platformType) {
+      return platformType.key == value;
+    }, Constants.platformTypes), 'name');
+  },
+  
+
+  updateFromTemplateOrDesign: function(design, template, idProperty, source, displayProperty, setReadOnly, setData) {
+    var id = null;
+    if (design) {
+      id = design[idProperty];
+    } else if (template) {
+      id = template[idProperty];
+    }
+    if (id) {
+      var change = Utils.array.findFirstOrNull(Utils.array.idPredicate(id), source);
+      if (change) {
+        setData(change[displayProperty]);
+      }
+    }
+    setReadOnly(design || (template && template.idProperty));
+  },
+  
 };
 
 HotTarget = {};
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "HotTarget" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2703)